### PR TITLE
New package: py-deepsig

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

**Review by adamjstewart — DISMISSED** [review]
Did you report the PyPI tarball issue to the developers? Might want to make the global URL download from GitHub, otherwise every version will have to have a version-specific URL unless the devs fix this in the future.

**Comment by alalazo** [comment]
@spackbot run pipeline

**Comment by adamjstewart** [comment]
> Did you report the PyPI tarball issue to the developers? Might want to make the global URL download from GitHub, otherwise every version will have to have a version-specific URL unless the devs fix this in the future.

Still have this question

**Comment by qwertos** [comment]
Discussing with them now.

File: var/spack/repos/builtin/packages/py-deepsig/package.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+from spack import *
+
+
+class PyDeepsig(PythonPackage):
+    """deep-significance: Easy and Better Significance Testing for Deep Neural
+    Networks"""
+
+    homepage = "https://github.com/Kaleidophon/deep-significance"
+    pypi     = "deepsig/deepsig-1.2.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-deepsig/package.py]:
```suggestion
    pypi     = "deepsig/deepsig-1.2.1.tar.gz"
```

File: var/spack/repos/builtin/packages/py-deepsig/package.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    depends_on('py-pandas@1.3.3',       type=('build', 'run'))
+    depends_on('py-dill@0.3.4',         type=('build', 'run'))
+
+    def patch(self):
+        filter_file(r'^with open\("README_RAW.md", "r"\) as fh:',
+                    '# no op',
+                    'setup.py')
+        filter_file(r'^    long_description = fh.read\(\)',
+                    'long_description = ""',
+                    'setup.py')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-deepsig/package.py]: Probably simpler to do:
```suggestion
        filter_file('README_RAW.md', 'README.md', 'setup.py', string=True)
```
Alternatively:
```suggestion
        copy('README.md', 'README_RAW.md')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — DISMISSED** [review]

**Comment by qwertos** [comment]
@spackbot run pipeline

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**